src/utils/url.js

```diff
@@ -52,14 +52,9 @@ export const isTopicLink = (url: string, realm: string): boolean => {
   );
 };
 
-export const isGroupLink = (url: string, realm: string): boolean => {
-  const paths = getPathsFromUrl(url, realm);
-  return (
-    isUrlInAppLink(url, realm)
-    && ((paths.length === 2 && paths[0] === 'pm-with')
-      || (paths.length === 4 && paths[0] === 'pm-with' && paths[2] === 'near'))
-  );
-};
+export const isGroupLink = (paths: string[]): boolean =>
```

Comment:
Hmm, that makes the function very confusing. 

Sure you've saved the `getPathsFromUrl` and `isUrlInAppLink` but the function makes no sense as I would not expect the question `Is this param a link to a group` be asked on a string array.

Reply:
I could inline the function, but perhaps just a name change, to `hasGroupLink` or something like that be better?